docs/manual/related-projects.md

```diff
@@ -20,6 +20,11 @@ a tool to snapshot systems on the client side (dpkg/rpm + BTRFS/LVM),
 or a tool to compose on a server and replicate (ChromiumOS, Clear
 Linux).  OSTree is flexible enough to do both.
 
+Note that this section of the docuementation is almost entirely
```

Comment:
Actually, it looks like if I go to that branch on your repo, I can directly edit files from the UI. Will try that next time.